portfolio.go

```diff
@@ -122,3 +122,41 @@ func (c *Client) ConvertPosition(positionParams ConvertPositionParams) (bool, er
 
 	return b, err
 }
+
+// HoldingsAuthParams represents the input params for initiating holdings authorization
+type HoldingsAuthParams struct {
+	ISIN     string
+	Quantity float64
```

Comment:
There are more params like 

`type`, `transfer_type`, `counter_demat`, `exec_date` so this struct had to include those and have another struct to hold `isin` and `quantity` fields which can be repeated.

Something like below 

```
type HoldingsAuthIns struct {
	ISIN     string
	Quantity float64
}

type HoldingsAuthParams struct {
   Ins []HoldingsAuthIns
   ....
   ...
}
```

Reply:
Added these fields